packages/@aws-cdk/aws-events-targets/lib/ecs-task.ts

```diff
@@ -104,6 +114,7 @@ export class EcsTask implements events.IRuleTarget {
     this.cluster = props.cluster;
     this.taskDefinition = props.taskDefinition;
     this.taskCount = props.taskCount !== undefined ? props.taskCount : 1;
+    this.platformVersion = props.platformVersion !== undefined ? props.platformVersion : ecs.FargatePlatformVersion.LATEST;
```

Comment:
```suggestion
    this.platformVersion = props.platformVersion ?? ecs.FargatePlatformVersion.LATEST;
```

Why do we need to set the platform version here to `LATEST` if unset?
Where is it currently implicitly set to `LATEST`?

Reply:
[this document](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-events-rule-ecsparameters.html#cfn-events-rule-ecsparameters-platformversion) describes:

> PlatformVersion
> Specifies the platform version for the task. Specify only the numeric portion of the platform version, such as 1.1.0. If you omit this, the latest platform version is used.

And, I found that in some places I set `LATEST` to the default version.
for example:

- https://github.com/aws/aws-cdk/blob/238d96380c10586c141691a215c5e74a44091642/packages/%40aws-cdk/aws-ecs-patterns/lib/fargate/queue-processing-fargate-service.ts#L59
- https://github.com/aws/aws-cdk/blob/238d96380c10586c141691a215c5e74a44091642/packages/%40aws-cdk/aws-ecs-patterns/lib/fargate/application-load-balanced-fargate-service.ts#L75
- https://github.com/aws/aws-cdk/blob/238d96380c10586c141691a215c5e74a44091642/packages/%40aws-cdk/aws-ecs/lib/fargate/fargate-service.ts#L57

Reply:
these examples are outside of this construct library (events-targets). why not allow the module to set the default? i'm unclear on the need to set it explicitly ourselves. what happens if it's left undefined? does it default to `LATEST` anyways?

Reply:
OK. I reconsidered.
I'll leave this default value to the ecs and ecs-patterns settings, and event-targets will not have a default value.